assets/src/models/routeTab.ts

```diff
@@ -38,11 +56,18 @@ export const parseRouteTabData = (
 ): RouteTab[] => {
   return routeTabsData.map((routeTabData) => ({
     id: routeTabData.id,
-    ordering: routeTabData.ordering,
-    presetName: routeTabData.preset_name,
+    ordering: nullToUndefined(routeTabData.ordering),
+    presetName: nullToUndefined(routeTabData.preset_name),
     isCurrentTab: routeTabData.is_current_tab || false,
     selectedRouteIds: routeTabData.selected_route_ids,
     ladderDirections: routeTabData.ladder_directions,
     ladderCrowdingToggles: routeTabData.ladder_crowding_toggles,
   }))
 }
+
+export const isPreset = (routeTab: RouteTab): boolean =>
+  routeTab.ordering === undefined
+export const isNotPreset = (routeTab: RouteTab): boolean => !isPreset(routeTab)
+
+const nullToUndefined = <T>(data: T | null): T | undefined =>
+  data === null ? undefined : data
```

Comment:
**suggestion (non-blocking):** it might be nice to put this in a module under the `helpers` directory, though I don't think it really fits into any of the existing modules.